### PR TITLE
DOCS-7578 Add NetFlow Monitor Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Adds a doc page for NetFlow Monitors in the Monitors and NetFlow Monitoring sections.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/monitors/types/netflow.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: NetFlow Monitor
+kind: Documentation
+further_reading:
+- link: "/network_monitoring/devices/netflow/"
```
Comment (alai97) [inline content/en/monitors/types/netflow.md:5]:
```suggestion
- link: "/network_monitoring/devices/netflow/#visualization"
```

File: content/en/network_monitoring/devices/netflow.md [hunk 2]
```diff
@@ -83,15 +85,126 @@ From the **Configuration** tab in NetFlow, click **Add Enrichment** to upload th
...
+| Device IP | IP address used to map to a device in NDM for enrichment purposes. |
+| Exporter IP | IP address from which NetFlow packets originate. |
+| Device Model | Model of the device. |
+| Device Name | Name of the device. |
+| Device Namespace | Namespace of the device. |
+| Device Vendor | Vendor of the device. |
+
+### Flow Details
+
+Represents characteristics of the network flow.
```
Comment (cswatt) [inline content/en/network_monitoring/devices/netflow.md:122]:
```suggestion
The following fields represent characteristics of the network flow.
```

File: content/en/network_monitoring/devices/netflow.md [hunk 3]
```diff
@@ -83,15 +85,126 @@ From the **Configuration** tab in NetFlow, click **Add Enrichment** to upload th
...
+| End Time | Timestamp of the last network packet between the source and destination IP addresses. |
+| Ether Type | Type of Ethernet frame encapsulation (IPv4 or IPv6). |
+| Flow Type | Type of NetFlow data format (IPFIX, sFlow5, NetFlow5, NetFlow9, or Unknown). |
+| IP Protocol | Protocol used for communication (such as ICMP, TCP, or UDP). |
+| Next Hop IP | IP address of the next hop in the network path. |
+| TCP Flag | Union of all TCP flags observed over the life of the flow. |
+| Bytes | Total number of bytes transferred. |
+| Packets | Total number of packets transferred. |
+
+Use the out-of-the-box facets to start analyzing traffic patterns based on NetFlow destination and source IP addresses.
```
Comment (cswatt) [inline content/en/network_monitoring/devices/netflow.md:137]: Is this line in the right place? Should it be under a facets-related section? When placed here, it looks like it pertains to the Flow Details section

File: content/en/network_monitoring/devices/netflow.md [hunk 4]
```diff
@@ -83,15 +85,126 @@ From the **Configuration** tab in NetFlow, click **Add Enrichment** to upload th
...
+| Source Port | The source port number. |
+| Source Subdivision ISO Code | The ISO code representing the subdivision (such as state or province) associated with the source IP. |
+| Source Subdivision Name | The name of the subdivision (such as state or province) associated with the source IP. |
+| Source Timezone | The timezone associated with the source IP. |
+
+By monitoring these key fields in NetFlow events, organizations can gain visibility into their network infrastructure, optimize performance, and improve security posture.
+
+{{< img src="monitors/monitor_types/netflow/monitor.png" alt="Create a dashboard with NetFlow data" width="100%" >}}
+
+This data is also available in dashboards, notebooks, and more for more precise queries and for correlating with other sources of data. When creating a dashboard with Netflow data, select **NetFlow** as the source in the **Graph your data** section.
```
Comment (cswatt) [inline content/en/network_monitoring/devices/netflow.md:199]: Can we reword this to avoid the awkward `more for more` construction? (also, don't forget to capitalize the F in NetFlow)

**Review by cswatt — CHANGES_REQUESTED** [review]
Looks great overall, my suggestions are pretty minor!

File: content/en/monitors/types/netflow.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## Overview
+
+Datadog [Network Device Monitoring][1] (NDM) provides visibility into your on-premises and virtual network devices, such as routers, switches, and firewalls. As a part of NDM, [NetFlow Monitoring][2] to examine, interpret, and analyze your network traffic flow data over time, and identify top contributors to network congestion. 
+
+After enabling NetFlow Monitoring, you can create a NetFlow monitor to alert you when a flow metric such as a monitoring network throughput for a specific source or destination pair crosses a threshold that you have set.
+
+## Monitor creation
+
+To create an NPM monitor in Datadog, use the main navigation: [**Monitors** --> **New Monitor** --> **NetFlow**][3].
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/monitors/types/netflow.md:31]: Good catch, thanks!

File: content/en/network_monitoring/devices/netflow.md (hunk 3 above)
Comment (alai97) [reply content/en/network_monitoring/devices/netflow.md:137]:
```suggestion
In addition to fields, you can also use out-of-the-box facets to start analyzing traffic patterns based on NetFlow destination and source IP addresses.
```

File: content/en/network_monitoring/devices/netflow.md [hunk 6]
```diff
@@ -83,15 +85,126 @@ From the **Configuration** tab in NetFlow, click **Add Enrichment** to upload th
...
+| Source Latitude | The latitude coordinate associated with the source IP. |
+| Source Longitude | The longitude coordinate associated with the source IP. |
+| Source MAC | The Media Access Control (MAC) address associated with the source IP. |
+| Source Mask | The subnet mask associated with the source IP. |
+| Source Port | The source port number. |
+| Source Subdivision ISO Code | The ISO code representing the subdivision (such as state or province) associated with the source IP. |
+| Source Subdivision Name | The name of the subdivision (such as state or province) associated with the source IP. |
+| Source Timezone | The timezone associated with the source IP. |
+
+By monitoring these key fields in NetFlow events, organizations can gain visibility into their network infrastructure, optimize performance, and improve security posture.
```
Comment (alai97) [inline content/en/network_monitoring/devices/netflow.md:195]:
```suggestion
By monitoring these key fields and using facets in NetFlow events, organizations can gain visibility into their network infrastructure, optimize performance, and improve security posture.
```

File: content/en/network_monitoring/devices/netflow.md (hunk 4 above)
Comment (alai97) [reply content/en/network_monitoring/devices/netflow.md:199]:
```suggestion
This data is also available in dashboards and notebooks, enabling precise queries and correlation with other data sources. When creating a dashboard with NetFlow data, select **NetFlow** as the source in the **Graph your data** section.
```

**Review by cswatt — APPROVED** [review]
